hack/make-rules/clean-local-cluster.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 The KubeAdmiral Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if [[ ! $(kind get clusters | grep kubeadmiral ) ]]; then
+  echo "There is no local kind cluster related with kubeadmiral."
+else
+  kind delete clusters $(kind get clusters | grep kubeadmiral | tr -s "\r\n" " ")
+  echo "The following files will be deleted:
+  ${HOME}/.kube/kubeadmiral.config
+  ${HOME}/.kube/members.config
+  ${HOME}/.kube/kubeadmiral"
+  rm -rf ${HOME}/.kube/kubeadmiral.config ${HOME}/.kube/members.config ${HOME}/.kube/kubeadmiral
```

Comment:
why do we need the `-f` here?

Reply:
Sometimes the file to be deleted may not exist, `-f` can skip the non-existing file without displaying any error message.